lang_interface/java/com/intel/daal/utils/LibUtils.java

```diff
@@ -48,24 +51,40 @@
      */
     public static void loadLibrary()
     {
-        try {
-            logger.log(logLevel, "Loading library " + DAALLIB + " as file.");
-            System.loadLibrary(DAALLIB);
-            logger.log(logLevel, "DONE: Loading library " + DAALLIB + " as file.");
-            return;
-        }
-        catch (UnsatisfiedLinkError e) {
-            logger.log(logLevel, "Can`t find library " + DAALLIB + " in java.library.path.");
-        }
+	if(loaded){
```

Comment:
not initialized variable usage

Reply:
In Java, the default value of boolean variable is false. So no need to initialize the "loaded" explicitly.